src/ripple/app/ledger/LedgerMaster.h

```diff
@@ -422,12 +422,18 @@ class LedgerMaster : public Stoppable, public AbstractFetchPackContainer
                   collector->make_gauge("LedgerMaster", "Validated_Ledger_Age"))
             , publishedLedgerAge(
                   collector->make_gauge("LedgerMaster", "Published_Ledger_Age"))
+            , outboundFetchPackRequests( 
+                  collector->make_counter("LedgerMaster", "Outbound_Fetch_Pack_Requests"))
+            , inboundFetchPackRequests(
+                  collector->make_counter("LedgerMaster", "Inbound_Fetch_Pack_Requests"))
```

Comment:
I almost missed this. clang-format wants to wrap these lines: https://github.com/ripple/rippled/pull/3555/checks?check_run_id=883715497. Please fix.

Reply:
Thanks for catching